views/block.tmpl

```diff
@@ -209,6 +210,9 @@
                                     </span>
                                 </td>
                                 <td class="mono fs15">{{.VoteInfo.Version}}</td>
+                                {{with .VoteInfo}}
+                                    <td>{{.Validation.Validity}}</td>
```

Comment:
Why not `.VoteInfo.Validation.Validity`?

Reply:
It was just a choice i made, but i think .VoteInfo.Validation.Validity would be a much cleaner way to write it. I'll change it.

Reply:
OK, it's just that in general I find it's good to use fewer lines if there's no reason not to